### ISSUE TITLE
Статыстыка

### ISSUE THREAD

**Issue by SkyfaceD** [task-management-system/backend#43]

**Comment by SkyfaceD** [comment]
@Seasle 

Гет запрос localhost:7000/api/v1/statistic

Ответ:
```json
{
    "message": null,
    "data": {
        "task": {
            "all": {
                "new": 16,
                "inWork": 2,
                "canceled": 0,
                "closed": 0
            },
            "actual": {
                "new": 10,
                "inWork": 1,
                "canceled": 0,
                "closed": 0
            }
        },
        "disk": {
            "available": 278271643648,
            "used": 233161859072
        }
    }
}
```

**Comment by SkyfaceD** [comment]
@Seasle breaking changes!1! https://github.com/task-management-system/backend/commits/dev
`localhost:7000/api/v1/statistic` -> `localhost:7000/api/v1/statistics`
`statistic` -> `statistics`
